pkg/cloudprovider/aws/factory.go

```diff
@@ -51,6 +54,7 @@ type Factory struct {
 
 func NewFactory(options cloudprovider.Options) *Factory {
 	sess := withRegion(session.Must(session.NewSession(&aws.Config{STSRegionalEndpoint: endpoints.RegionalSTSEndpoint})))
+	sess = withUserAgent(sess)
```

Comment:
Can you inline this?

Reply:
yeah, was debating whether to or not, should've known to always go with the inline :D